[2024-07-26T00:01:12.289+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-26T00:01:12.306+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_script_dag.run_elt_script scheduled__2024-07-25T00:00:00+00:00 [queued]>
[2024-07-26T00:01:12.312+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_script_dag.run_elt_script scheduled__2024-07-25T00:00:00+00:00 [queued]>
[2024-07-26T00:01:12.313+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-07-26T00:01:12.323+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): run_elt_script> on 2024-07-25 00:00:00+00:00
[2024-07-26T00:01:12.329+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=1481) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-26T00:01:12.331+0000] {standard_task_runner.py:64} INFO - Started process 1483 to run task
[2024-07-26T00:01:12.330+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'elt_script_dag', 'run_elt_script', 'scheduled__2024-07-25T00:00:00+00:00', '--job-id', '119', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpyqs4bs92']
[2024-07-26T00:01:12.332+0000] {standard_task_runner.py:91} INFO - Job 119: Subtask run_elt_script
[2024-07-26T00:01:12.366+0000] {task_command.py:426} INFO - Running <TaskInstance: elt_script_dag.run_elt_script scheduled__2024-07-25T00:00:00+00:00 [running]> on host 6a3ef3606308
[2024-07-26T00:01:12.432+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt_script_dag' AIRFLOW_CTX_TASK_ID='run_elt_script' AIRFLOW_CTX_EXECUTION_DATE='2024-07-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-25T00:00:00+00:00'
[2024-07-26T00:01:12.434+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-26T00:01:12.435+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-26T00:01:12.436+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/scripts/elt_script.py']
[2024-07-26T00:01:12.447+0000] {subprocess.py:86} INFO - Output:
[2024-07-26T00:01:15.591+0000] {subprocess.py:93} INFO - ELT script started...
[2024-07-26T00:01:15.592+0000] {subprocess.py:93} INFO - fetching videos...
[2024-07-26T00:01:15.592+0000] {subprocess.py:93} INFO - page flipped...
[2024-07-26T00:01:15.593+0000] {subprocess.py:93} INFO - page flipped...
[2024-07-26T00:01:15.593+0000] {subprocess.py:93} INFO - page flipped...
[2024-07-26T00:01:15.593+0000] {subprocess.py:93} INFO - videos uploaded to the db.
[2024-07-26T00:01:15.594+0000] {subprocess.py:93} INFO - No. of channels to be fetched:  14
[2024-07-26T00:01:15.594+0000] {subprocess.py:93} INFO - fetching channel data...
[2024-07-26T00:01:15.594+0000] {subprocess.py:93} INFO - flipping page...
[2024-07-26T00:01:15.595+0000] {subprocess.py:93} INFO - Error inserting data: column "video_count" of relation "channels" does not exist
[2024-07-26T00:01:15.595+0000] {subprocess.py:93} INFO - LINE 1: ...username, descr, channel_start_date, subs, views, video_coun...
[2024-07-26T00:01:15.595+0000] {subprocess.py:93} INFO -                                                              ^
[2024-07-26T00:01:15.595+0000] {subprocess.py:93} INFO - 
[2024-07-26T00:01:15.596+0000] {subprocess.py:93} INFO - Error inserting data: column "video_count" of relation "channels" does not exist
[2024-07-26T00:01:15.596+0000] {subprocess.py:93} INFO - LINE 1: ...username, descr, channel_start_date, subs, views, video_coun...
[2024-07-26T00:01:15.596+0000] {subprocess.py:93} INFO -                                                              ^
[2024-07-26T00:01:15.597+0000] {subprocess.py:93} INFO - 
[2024-07-26T00:01:15.597+0000] {subprocess.py:93} INFO - Error inserting data: column "video_count" of relation "channels" does not exist
[2024-07-26T00:01:15.597+0000] {subprocess.py:93} INFO - LINE 1: ...username, descr, channel_start_date, subs, views, video_coun...
[2024-07-26T00:01:15.598+0000] {subprocess.py:93} INFO -                                                              ^
[2024-07-26T00:01:15.598+0000] {subprocess.py:93} INFO - 
[2024-07-26T00:01:15.598+0000] {subprocess.py:93} INFO - Error inserting data: column "video_count" of relation "channels" does not exist
[2024-07-26T00:01:15.599+0000] {subprocess.py:93} INFO - LINE 1: ...username, descr, channel_start_date, subs, views, video_coun...
[2024-07-26T00:01:15.599+0000] {subprocess.py:93} INFO -                                                              ^
[2024-07-26T00:01:15.599+0000] {subprocess.py:93} INFO - 
[2024-07-26T00:01:15.600+0000] {subprocess.py:93} INFO - Error inserting data: column "video_count" of relation "channels" does not exist
[2024-07-26T00:01:15.600+0000] {subprocess.py:93} INFO - LINE 1: ...username, descr, channel_start_date, subs, views, video_coun...
[2024-07-26T00:01:15.600+0000] {subprocess.py:93} INFO -                                                              ^
[2024-07-26T00:01:15.600+0000] {subprocess.py:93} INFO - 
[2024-07-26T00:01:15.601+0000] {subprocess.py:93} INFO - Error inserting data: column "video_count" of relation "channels" does not exist
[2024-07-26T00:01:15.601+0000] {subprocess.py:93} INFO - LINE 1: ...username, descr, channel_start_date, subs, views, video_coun...
[2024-07-26T00:01:15.601+0000] {subprocess.py:93} INFO -                                                              ^
[2024-07-26T00:01:15.602+0000] {subprocess.py:93} INFO - 
[2024-07-26T00:01:15.602+0000] {subprocess.py:93} INFO - Error inserting data: column "video_count" of relation "channels" does not exist
[2024-07-26T00:01:15.602+0000] {subprocess.py:93} INFO - LINE 1: ...username, descr, channel_start_date, subs, views, video_coun...
[2024-07-26T00:01:15.602+0000] {subprocess.py:93} INFO -                                                              ^
[2024-07-26T00:01:15.603+0000] {subprocess.py:93} INFO - 
[2024-07-26T00:01:15.603+0000] {subprocess.py:93} INFO - Error inserting data: column "video_count" of relation "channels" does not exist
[2024-07-26T00:01:15.603+0000] {subprocess.py:93} INFO - LINE 1: ...username, descr, channel_start_date, subs, views, video_coun...
[2024-07-26T00:01:15.603+0000] {subprocess.py:93} INFO -                                                              ^
[2024-07-26T00:01:15.604+0000] {subprocess.py:93} INFO - 
[2024-07-26T00:01:15.604+0000] {subprocess.py:93} INFO - Error inserting data: column "video_count" of relation "channels" does not exist
[2024-07-26T00:01:15.604+0000] {subprocess.py:93} INFO - LINE 1: ...username, descr, channel_start_date, subs, views, video_coun...
[2024-07-26T00:01:15.605+0000] {subprocess.py:93} INFO -                                                              ^
[2024-07-26T00:01:15.605+0000] {subprocess.py:93} INFO - 
[2024-07-26T00:01:15.605+0000] {subprocess.py:93} INFO - Error inserting data: column "video_count" of relation "channels" does not exist
[2024-07-26T00:01:15.606+0000] {subprocess.py:93} INFO - LINE 1: ...username, descr, channel_start_date, subs, views, video_coun...
[2024-07-26T00:01:15.606+0000] {subprocess.py:93} INFO -                                                              ^
[2024-07-26T00:01:15.606+0000] {subprocess.py:93} INFO - 
[2024-07-26T00:01:15.607+0000] {subprocess.py:93} INFO - Error inserting data: column "video_count" of relation "channels" does not exist
[2024-07-26T00:01:15.607+0000] {subprocess.py:93} INFO - LINE 1: ...username, descr, channel_start_date, subs, views, video_coun...
[2024-07-26T00:01:15.607+0000] {subprocess.py:93} INFO -                                                              ^
[2024-07-26T00:01:15.608+0000] {subprocess.py:93} INFO - 
[2024-07-26T00:01:15.608+0000] {subprocess.py:93} INFO - Error inserting data: column "video_count" of relation "channels" does not exist
[2024-07-26T00:01:15.608+0000] {subprocess.py:93} INFO - LINE 1: ...username, descr, channel_start_date, subs, views, video_coun...
[2024-07-26T00:01:15.608+0000] {subprocess.py:93} INFO -                                                              ^
[2024-07-26T00:01:15.609+0000] {subprocess.py:93} INFO - 
[2024-07-26T00:01:15.609+0000] {subprocess.py:93} INFO - Done.
[2024-07-26T00:01:15.617+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-26T00:01:15.617+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-26T00:01:15.637+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=elt_script_dag, task_id=run_elt_script, run_id=scheduled__2024-07-25T00:00:00+00:00, execution_date=20240725T000000, start_date=20240726T000112, end_date=20240726T000115
[2024-07-26T00:01:15.683+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-26T00:01:15.700+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-26T00:01:15.702+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
